Indicators/Alpha.cs

```diff
@@ -0,0 +1,260 @@
+        /// Risk free rate of the target used in relation with the reference
+        /// </summary>
+        private readonly decimal _riskFreeRate;
+
+        /// <summary>
+        /// Interest rate provider is used to automaticalle get the risk free rate of the target used in relation with the reference
+        /// </summary>
+        private readonly InterestRateProvider _interestRateProvider;
+
+        /// <summary>
+        /// Required period, in data points, for the indicator to be ready and fully initialized.
+        /// </summary>
+        public int WarmUpPeriod { get; private set; }
+
+        /// <summary>
+        /// Gets a flag indicating when the indicator is ready and fully initialized
+        /// </summary>
+        public override bool IsReady => _targetROC.IsReady && _beta.IsReady && _referenceROC.IsReady;
+
+        /// <summary>
+        /// Creates a new Alpha indicator with the specified name, target, reference, and period values
+        /// </summary>
+        /// <param name="name">The name of this indicator</param>
+        /// <param name="targetSymbol">The target symbol of this indicator</param>
+        /// <param name="referenceSymbol">The reference symbol of this indicator</param>
+        /// <param name="alphaPeriod">Period of the indicator - alpha</param>
+        /// <param name="betaPeriod">Period of the indicator - beta</param>
+        /// <param name="riskFreeRate">The risk free rate of this indicator for given period</param>
+        /// <param name="useInterestRateProvider">Use interest rate provider to automatically get the risk free rate of this indicator for given period</param>
+        public Alpha(string name, Symbol targetSymbol, Symbol referenceSymbol, int alphaPeriod, int betaPeriod, decimal riskFreeRate = 0.0m, bool useInterestRateProvider = false)
+            : base(name)
+        {
+            // Assert that the target and reference symbols are not the same
+            if (targetSymbol == referenceSymbol)
+            {
+                throw new ArgumentException("The target and reference symbols cannot be the same.");
+            }
+
+            // Assert that the period is greater than 2, otherwise the alpha can not be computed
+            if (alphaPeriod < 1)
+            {
+                throw new ArgumentException("The period must be equal or greater than 1.");
+            }
+
+            // Assert that the beta period is greater than 2, otherwise the beta can not be computed
+            if (betaPeriod < 2)
+            {
+                throw new ArgumentException("The beta period must be equal or greater than 2.");
+            }
+            
+            _targetSymbol = targetSymbol;
+            _referenceSymbol = referenceSymbol;
+            _alphaPeriod = alphaPeriod;
+            _betaPeriod = betaPeriod;
+
+            if (useInterestRateProvider)
+            {
+                _interestRateProvider = new InterestRateProvider();
+            }
```

Comment:
Hey let's go back to only having the `riskFreeRate` parameter. There is work in line that will make this not useful quite soon. The pattern in similar indicators is just having the `riskFreeRate` parameter.